### PR TITLE
[doc] update installation doc

### PR DESCRIPTION
**Pull request by GuillaumeEscande**
# PLEASE READ ALL ITEMS AND CHECK ONLY RELEVANT CHECKBOXES BELOW

## Project management

- [] Has the pull request been added to the relevant milestone?
- [x] Have the `priority:` and `pr:` labels been added to the pull request? (In case of doubt, start with the labels `priority: low` and `pr: to review later`)
- [ ] Have the relevant issues been added to the pull request?
- [ ] Have the relevant labels been added to the issues? (`area:`, `type:`)
- [ ] Have the relevant issues been added to the same project milestone as the pull request?

## Changelog and release notes

- [ ] Has the `CHANGELOG.adoc` + `doc/content/modules/user-manual/pages/release-notes/YYYY.MM.0.adoc` been updated to reference the relevant issues?
- [ ] Have the relevant API breaks been described in the `CHANGELOG.adoc` + `doc/content/modules/user-manual/pages/release-notes/YYYY.MM.0.adoc`?
- [ ] In case of a change with a visual impact, are there any screenshots in the `doc/content/modules/user-manual/pages/release-notes/YYYY.MM.0.adoc`?
- [ ] In case of a key change, has the change been added to `Key highlights` section in `doc/content/modules/user-manual/pages/release-notes/YYYY.MM.0.adoc`?
- [ ] Are the new / upgraded dependencies mentioned in the relevant section of the `CHANGELOG.adoc` + `doc/content/modules/user-manual/pages/release-notes/YYYY.MM.0.adoc`?

## Documentation

- [x] Have you included an update of the [documentation](https://doc.mbse-syson.org) in your pull request? Please ask yourself if an update (installation manual, user manual, developer manual...) is needed and add one accordingly.

## Tests

- [ ] Is the code properly tested? Any pull request (fix, enhancement or new feature) should come with a test (or several). It could be unit tests, integration tests or cypress tests depending on the context. Only doc and releng pull request do not need for tests.

### CODE REVIEW

File: doc/content/modules/installation-guide/pages/how-tos/install/production_deploy.adoc [hunk 1]
```diff
@@ -0,0 +1,102 @@
+= How to install {product} for production ?
+
+This section provides instructions for installing a production-ready version of the application. This setup is intended for deployment in a live environment, with full multi-user access and all necessary security configurations.
```
Comment (mbats) [inline doc/content/modules/installation-guide/pages/how-tos/install/production_deploy.adoc:3]: One sentence per line

File: doc/content/modules/installation-guide/pages/how-tos/install/production_deploy.adoc [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+Only `YEAR.MONTH.0` versions are stable and suitable for production deployments.
+Other versions, such as intermediate releases, are intended solely for testing purposes.
+They're not recommended for production use due to their potential for further changes and instability.
+Always use the stable `.0` releases for any production environments to ensure reliability and support.
+For more details see xref:user-manual:cycle.adoc[].
+====
+
+== Prepare ecosystem
+
+Pay attention to deploy enterprise grade version of the prerequisites. In particular the databse service. Before do deploy manually the databse, we recomend to use managed database instance (provided by the IT service of a corporation or using cloud services)
```
Comment (mbats) [inline doc/content/modules/installation-guide/pages/how-tos/install/production_deploy.adoc:19]: One sentence per line

Comment (mbats) [reply doc/content/modules/installation-guide/pages/how-tos/install/production_deploy.adoc:19]: Pay attention -> Be sure
enterprise grade -> enterprise-grade
of the prerequisites. In particular the databse service. -> of the prerequisites, especially the database service. 
Before do deploy -> Before manually deploying
the databse -> the database
recomend -> recommend
to use managed database instance -> using a managed database instance
(provided by the IT service of a corporation or using cloud services) -> , which can be provided by your corporation's IT service or through cloud services.

File: doc/content/modules/installation-guide/pages/how-tos/install/production_deploy.adoc [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+[INFO]
+====
+Before you proceed with the installation, please ensure you have the following xref:requirements.adoc[prerequisites]:
+====
+
+We recommand to provide a dedicated host (server or virtual machine) to serve the syson application and only the application.
```
Comment (mbats) [inline doc/content/modules/installation-guide/pages/how-tos/install/production_deploy.adoc:26]: recommand -> recommend
to provide -> providing
(server or virtual machine) -> , either a server or a virtual machine,
to serve -> to run
syson -> {product}
and only the application -> exclusively

File: doc/content/modules/installation-guide/pages/how-tos/install/production_deploy.adoc [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+
+This step provide the following configuration elements :
```
Comment (mbats) [inline doc/content/modules/installation-guide/pages/how-tos/install/production_deploy.adoc:28]: provide -> provides

File: doc/content/modules/installation-guide/pages/how-tos/install/production_deploy.adoc [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+.Stable versions
+====
+Only `YEAR.MONTH.0` versions are stable and suitable for production deployments.
+Other versions, such as intermediate releases, are intended solely for testing purposes.
+They're not recommended for production use due to their potential for further changes and instability.
+Always use the stable `.0` releases for any production environments to ensure reliability and support.
+For more details see xref:user-manual:cycle.adoc[].
+====
+
+== Prepare ecosystem
```
Comment (mbats) [inline doc/content/modules/installation-guide/pages/how-tos/install/production_deploy.adoc:17]: Preparing the Ecosystem for Deployment

File: doc/content/modules/installation-guide/pages/how-tos/install/production_deploy.adoc [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+. Click the latest version to open its specific page.
+. In the `Assets` section, locate the JAR file named `{product-jar}`.
++
+image::download.png[Download JAR file]
+. Click the JAR file to start the download to your local machine.
+
+This process ensures that you get the most recent version of the {product} application in the form of a pre-built JAR file.
+
+[#secure-app]
+== Secure the application
```
Comment (mbats) [inline doc/content/modules/installation-guide/pages/how-tos/install/production_deploy.adoc:57]: Secure -> Securing

File: doc/content/modules/installation-guide/pages/how-tos/install/production_deploy.adoc [hunk 7]
```diff
@@ -0,0 +1,102 @@
...
+
+As explain on this xref:installation-guide:how-tos/https.adoc[guide], the {product} application must be secured to ba availlable on a network. For a production grade deployment, certificates used to secure the connexion must be valid and provided by a valid authority.
```
Comment (mbats) [inline doc/content/modules/installation-guide/pages/how-tos/install/production_deploy.adoc:59]: On sentence per line

Comment (mbats) [reply doc/content/modules/installation-guide/pages/how-tos/install/production_deploy.adoc:59]: explain -> explained
on this -> in the
to ba availlable -> to be accessible

Comment (mbats) [reply doc/content/modules/installation-guide/pages/how-tos/install/production_deploy.adoc:59]: production grade -> production-grade
certificates -> the certificates
connexion -> connection
provided by a valid authority -> issued by a recognized authority

File: doc/content/modules/installation-guide/pages/how-tos/install/production_deploy.adoc [hunk 8]
```diff
@@ -0,0 +1,102 @@
...
+
+Documentation to generate this kind of certificate is provided by dedicated organism as exemple (https://www.namecheap.com/support/knowledgebase/article.aspx/9422/2290/generating-a-csr-on-tomcat-using-a-keytool/)[Nameship]
```
Comment (mbats) [inline doc/content/modules/installation-guide/pages/how-tos/install/production_deploy.adoc:61]: to generate -> for generating
kind -> type
organism -> organizations
as exemple -> such as
Nameship -> Namecheap

File: doc/content/modules/installation-guide/pages/how-tos/install/production_deploy.adoc [hunk 9]
```diff
@@ -0,0 +1,102 @@
...
+
+Following the Autority certification authority, a PKCS12 keystore must be produced.
```
Comment (mbats) [inline doc/content/modules/installation-guide/pages/how-tos/install/production_deploy.adoc:63]: the Autority certification -> the guidelines of the certification
produced -> created

File: doc/content/modules/installation-guide/pages/how-tos/install/production_deploy.adoc [hunk 10]
```diff
@@ -0,0 +1,102 @@
...
+
+This step provide the following configuration elements :
```
Comment (mbats) [inline doc/content/modules/installation-guide/pages/how-tos/install/production_deploy.adoc:65]: provide -> provides

File: doc/content/modules/installation-guide/pages/how-tos/install/production_deploy.adoc [hunk 11]
```diff
@@ -0,0 +1,102 @@
...
+
+* keystore path
+* keystore password
+* private key password
+* certificate alias name
+
+[#start-app]
+== Start the application
```
Comment (mbats) [inline doc/content/modules/installation-guide/pages/how-tos/install/production_deploy.adoc:73]: Start -> Starting

File: doc/content/modules/installation-guide/partials/nav.adoc [hunk 12]
```diff
@@ -4,6 +4,10 @@
 // Each navigation file must be declared in the component descriptor if you want it to be displayed in the component’s navigation menu.
 * xref:installation-guide:requirements.adoc[]
 * xref:installation-guide:how-tos/install.adoc[]
-* xref:installation-guide:how-tos/https.adoc[]
+** xref:installation-guide:how-tos/install/local_test.adoc[Simple local test of Syson]
```
Comment (mbats) [inline doc/content/modules/installation-guide/partials/nav.adoc:7]: Basic Local Test Setup

File: doc/content/modules/installation-guide/partials/nav.adoc [hunk 13]
```diff
@@ -4,6 +4,10 @@
...
+** xref:installation-guide:how-tos/install/multi_user_test.adoc[Local test of Syson with multiple users]
```
Comment (mbats) [inline doc/content/modules/installation-guide/partials/nav.adoc:8]: Multi-User Local Test Setup

File: doc/content/modules/installation-guide/partials/nav.adoc [hunk 14]
```diff
@@ -4,6 +4,10 @@
...
+** xref:installation-guide:how-tos/install/production_deploy.adoc[Production grade deployment]
```
Comment (mbats) [inline doc/content/modules/installation-guide/partials/nav.adoc:9]: Production-Grade Deployment

File: doc/content/modules/installation-guide/partials/nav.adoc [hunk 15]
```diff
@@ -4,6 +4,10 @@
...
+** xref:installation-guide:how-tos/install/ecosystem_only.adoc[Ecosystem only]
```
Comment (mbats) [inline doc/content/modules/installation-guide/partials/nav.adoc:10]: Ecosystem-Only Deployment

File: doc/content/modules/installation-guide/partials/nav.adoc [hunk 16]
```diff
@@ -4,6 +4,10 @@
...
+** xref:installation-guide:how-tos/https.adoc[Https configuration]
```
Comment (mbats) [inline doc/content/modules/installation-guide/partials/nav.adoc:11]: HTTPS

File: doc/content/modules/installation-guide/pages/how-tos/https.adoc [hunk 17]
```diff
@@ -10,6 +10,8 @@ For production deployments, obtain a valid certificate from a trusted CA to ensu
 If you intend to install {product} on a local server or machine and allow several users to access it, you must enable HTTPS.
 If you don't have a certificate, you can generate one by using https://docs.oracle.com/en/java/javase/11/tools/keytool.html[keytool] for testing purposes only.
 
+This procedure require to have the keytool application installed (the application is installed whith the Java Runtime Environment as explain in xref:requirements.adoc[prerequisites]:)
```
[the remark this answers is not in the document]
Comment (GuillaumeEscande) [reply doc/content/modules/installation-guide/pages/how-tos/https.adoc]: Done

File: doc/content/modules/installation-guide/pages/how-tos/install.adoc [hunk 18]
```diff
@@ -3,143 +3,9 @@
 This section delves into the detailed step-by-step instructions for installing {product}.
 Detailed explanations walk you through each step.
 
-== Get {product}
+Syson is installable following some different pattern :
```
[the remark this answers is not in the document]
Comment (GuillaumeEscande) [reply doc/content/modules/installation-guide/pages/how-tos/install.adoc]: Done

File: doc/content/modules/installation-guide/pages/how-tos/install.adoc [hunk 19]
```diff
@@ -3,143 +3,9 @@
...
-
-If you encounter any issues during the installation process, please refer to the xref:troubleshooting.adoc[troubleshooting guide] for detailed solutions.
-
-=== Other resources
-xref:installation-guide:how-tos/https.adoc[]
\ No newline at end of file
+* xref:installation-guide:how-tos/install/local_test.adoc[Simple local test of Syson]
+* xref:installation-guide:how-tos/install/multi_user_test.adoc[Local test of Syson with multiple users]
+* xref:installation-guide:how-tos/install/production_deploy.adoc[Production grade deployment]
+* xref:installation-guide:how-tos/install/ecosystem_only.adoc[Ecosystem only (for deployment purpose)]
```
[the remark this answers is not in the document]
Comment (GuillaumeEscande) [reply doc/content/modules/installation-guide/pages/how-tos/install.adoc:11]: Done

File: doc/content/modules/installation-guide/partials/nav.adoc [hunk 20]
```diff
@@ -4,6 +4,10 @@
 // Each navigation file must be declared in the component descriptor if you want it to be displayed in the component’s navigation menu.
 * xref:installation-guide:requirements.adoc[]
 * xref:installation-guide:how-tos/install.adoc[]
+** xref:installation-guide:how-tos/install/local_test.adoc[Simple local test of Syson]
+** xref:installation-guide:how-tos/install/multi_user_test.adoc[Local test of Syson with multiple users]
+** xref:installation-guide:how-tos/install/production_deploy.adoc[Production grade deployment]
+** xref:installation-guide:how-tos/install/ecosystem_only.adoc[Ecosystem only]
 * xref:installation-guide:how-tos/https.adoc[]
```
[the remark this answers is not in the document]
Comment (GuillaumeEscande) [reply doc/content/modules/installation-guide/partials/nav.adoc]: Done

File: doc/content/modules/installation-guide/partials/nav.adoc (hunk 12 above)
Comment (GuillaumeEscande) [reply doc/content/modules/installation-guide/partials/nav.adoc]: Done

File: doc/content/modules/installation-guide/partials/nav.adoc (hunk 13 above)
Comment (GuillaumeEscande) [reply doc/content/modules/installation-guide/partials/nav.adoc]: Done

File: doc/content/modules/installation-guide/partials/nav.adoc (hunk 15 above)
Comment (GuillaumeEscande) [reply doc/content/modules/installation-guide/partials/nav.adoc]: Done

File: doc/content/modules/installation-guide/partials/nav.adoc (hunk 16 above)
Comment (GuillaumeEscande) [reply doc/content/modules/installation-guide/partials/nav.adoc]: Done

File: doc/content/modules/installation-guide/pages/how-tos/install.adoc [hunk 21]
```diff
@@ -3,157 +3,9 @@
 This section delves into the detailed step-by-step instructions for installing {product}.
 Detailed explanations walk you through each step.
 
-== Get {product}
+{product} can be installed following several different patterns :
```
Comment (AxelRICHARD) [inline doc/content/modules/installation-guide/pages/how-tos/install.adoc:6]: In english, double punctuations don't need a space before.
Could you fix that here and everywhere else in this PR please?

**Pull request merged**